pages/common/composer.md

```diff
@@ -0,0 +1,23 @@
+# composer
+
+> A per-project package manager for PHP.
```

Comment:
It's not a project manager though. It manages dependencies on a project-by-project level.